setup.py

```diff
@@ -0,0 +1,59 @@
+# ============================================================
+#
+# Copyright (C) 2010-2018 by Johannes Wienke
+#
+# This file may be licensed under the terms of the
+# GNU Lesser General Public License Version 3 (the ``LGPL''),
+# or (at your option) any later version.
+#
+# Software distributed under the License is distributed
+# on an ``AS IS'' basis, WITHOUT WARRANTY OF ANY KIND, either
+# express or implied. See the LGPL for the specific language
+# governing rights and limitations.
+#
+# You should have received a copy of the LGPL along with this
+# program. If not, go to http://www.gnu.org/licenses/lgpl.html
+# or write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# ============================================================
+
+from setuptools import setup
+
+setup(name='rsb-spread-python',
+      version='1.0.0-dev',
+      description='Spread transport for rsb-python',
+      author='Johannes Wienke',
```

Comment:
Please add me here.

Reply:
There doesn't seem to be actual support for multiple authors or maintainers. Leaving out both is also not supported.

Reply:
Let's see whether a comma is ok